Lab 1/Lab1/LogsAndExceptions/FIOFieldException.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lab1.LogsAndExceptions
+{
+    public class FIOFieldException : Exception
```

Comment:
https://docs.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions